### ISSUE TITLE
Everpad inserts tab/spaces on edited notes

### ISSUE THREAD

**Issue by sasha42** [nvbn/everpad#412]
I keep finding that everpad seems to insert tabs or spaces quite often to a note that I frequently edit. What happens is that every line between the first and last line gets a tab/space upon every time everpad is relaunched (I don't think it's every time I edit the note), and the note can end up quite ugly in the end. I then have to manually delete these tabs to fix the note.

**Comment by julesbl** [comment]
Yes, I get this problem as well, seems to be something to do with editing the note on the android version of evernote, it looks right when I view the note but when I go into edit all these spaces and cariage returns appear. Fustrating
